RecoLocalMuon/CSCRecHitD/src/CSCRecHitDBuilder.cc

```diff
@@ -124,8 +125,15 @@ void CSCRecHitDBuilder::build(const CSCStripDigiCollection* stripdc,
     for (auto const& s_hit : cscStripHit) {
       for (auto const& w_hit : cscWireHit) {
         CSCRecHit2D rechit = make2DHits_->hitFromStripAndWire(sDetId, layer, w_hit, s_hit);
-
-        bool isInFiducial = make2DHits_->isHitInFiducial(layer, rechit);
+        // Store rechit as a Local Point:
+        LocalPoint rhitlocal = rechit.localPosition();
+        float yreco = rhitlocal.y();
+        bool isInFiducial = false;
+        if ((sDetId.station() == 1) && (sDetId.ring() == 1 || sDetId.ring() == 4) && (fabs(yreco + 30.) < 5.)) {
```

Comment:
some comment about the origin of 30 and 5 would be useful.
Is it the +-5 cm from the A/B boundary?

Reply:
I added the required comment. You understood the idea correctly.